app/src/operations/trigger/index.ts

```diff
@@ -34,6 +34,51 @@ export default defineOperationApp({
 					},
 				},
 			},
+			{
+				field: 'iterationMode',
+				name: '$t:operations.trigger.iteration_mode',
+				type: 'json',
+				meta: {
+					width: 'half',
+					interface: 'select-dropdown',
+					note: '$t:operations.trigger.iteration_mode_note',
+					options: {
+						choices: [
+							{
+								text: '$t:operations.trigger.serial',
+								value: 'serial',
+							},
+							{
+								text: '$t:operations.trigger.batch',
+								value: 'batch',
+							},
+							{
+								text: '$t:operations.trigger.parallel',
+								value: 'parallel',
+							},
+						],
+					},
+				},
+				schema: {
+					default_value: 'serial',
```

Comment:
Should we make `parallel` the default value as that is the current behavior?

Reply:
Yes 👍🏻